spec/lib/miq_automation_engine/models/miq_ae_yaml_import_export_spec.rb

```diff
@@ -197,8 +195,8 @@ def assert_import_failure_with_missing_file(import_options)
       end
 
       def assert_all_domains_imported(export_options, import_options)
-        export_model(ALL_DOMAINS, export_options)
-        reset_and_import(@export_dir, ALL_DOMAINS, import_options)
+        export_model(MiqAeYamlImportExportMixin::ALL_DOMAINS, export_options)
```

Comment:
> cue @jrafanie saying "RSpec's DSL is confusing...why can't we just use Ruby" ;)

I love that you know exactly what I mean... This entire PR is the perfect example of it.  rspec's mapping of examples/example groups to distinct test classes is a complete mystery and every time I figure it out, I forget the next time I need to know.  😕